requirements-opt.txt

```diff
@@ -1,3 +1,8 @@
 linkml-runtime==1.7.4; python_version >= "3.9"
 schemasheets==0.2.1; python_version >= "3.9"
 oaklib==0.5.32; python_version >= "3.9"
+
+# for streaming tests
+fsspec==2024.10.0
+requests==2.32.3
+aiohttp==3.10.10
```

Comment:
Are ``requests`` and ``aiohttp`` needed here or these being installed by fsspec? I don't see these libraries being used directly here and if ``fsspec`` installs them then I don't think we should need to specify these separately.

Reply:
fsspec does not install them. they are required for http streaming